Source/Adobe.Target.Client/Adobe.Target.Client.csproj

```
@@ -13,7 +13,7 @@
 
   <ItemGroup Label="Project References">
     <PackageReference Include="Adobe.ExperienceCloud.Ecid" Version="1.0.0" />
-    <PackageReference Include="Adobe.Target.Delivery" Version="1.1.3" />
+    <ProjectReference Include="../Adobe.Target.Delivery/Adobe.Target.Delivery.csproj" />
```

Let's not commit to Git local project reference, used when debugging:
```suggestion
    <ProjectReference Include="Adobe.Target.Delivery" Version="1.1.3" />
```